src/com/projectkorra/projectkorra/board/BendingBoardManager.java

```diff
@@ -118,29 +146,31 @@ public static void updateAllSlots(Player player) {
 
 	public static void updateBoard(Player player, String abilityName, boolean cooldown, int slot) {
```

Comment:
The parameter name `cooldown` and all of the subsequent places it gets passed to are sort of misleading. Something like `forceCooldown` would be a more appropriate name. Additionally, it is always `false` in our implementation. I think we should have wrappers to make it easier for us instead of always specifying an extraneous `false` all over the place.